File: Mixter/Domain/Subscriptions/UserUnfollowed.cs
namespace Mixter.Domain.Subscriptions
{
    public struct UserUnfollowed : IDomainEvent
    {
        public SubscriptionId SubscriptionId { get; private set; }

        public UserUnfollowed(SubscriptionId subscriptionId) : this()
        {
            SubscriptionId = subscriptionId;
        }
    }
}

File: app/Domain/Subscriptions/UserUnfollowed.php
<?php

namespace App\Domain\Subscriptions;

use App\Domain\IDomainEvent;

class UserUnfollowed implements IDomainEvent
{
    private $id;

    public function __construct(SubscriptionId $id)
    {
        $this->id = $id;
    }

    public function getAggregateId()
    {
        // TODO: Implement getAggregateId() method.
    }

    public function getSubscriptionId(): SubscriptionId
    {
        return $this->id;
    }
}
